cookbook/core/containerization/workflow_labels_annotations.py

```diff
@@ -1,8 +1,56 @@
 """
 Adding Workflow Labels and Annotations
----------------------------------------
+--------------------------------------
 
-.. NOTE::
+In Flyte, workflow executions are created as kubernetes resources. These can be extended with
+`labels <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_  and
+`annotations <https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/>`_.
 
-    Coming soon 🛠
-"""
\ No newline at end of file
+**Labels** and **annotations** are key value pairs which can be used to identify workflows for your own uses.
```

Comment:
@kumare3  do we need to add a warning to this section saying that annotations might trigger webhooks and such?  so be careful with what you add?

Reply:
Absolutely - GREAT callout!

Reply:
@wild-endeavor would you like to craft the warning?

Reply:
```
.. warning::
   Note that adding labels and annotations to your K8s resources may have side-effects depending on webhook behavior on your execution clusters.
```

Reply:
Thank you Yee, warning note added.